index/server/config/types.yaml

```diff
@@ -36,3 +36,7 @@ output-options:
       //
       // Code generated by {{.ModuleName}} version {{.Version}}. **DO NOT EDIT**
       package {{.PackageName}}
+
+      import (
+        "github.com/devfile/registry-support/index/generator/schema"
+      )
```

Comment:
Quick Question: Will there be any downsides to doing this? Since this will always be included when we generate is there any chance that k8s schema is also included and we run into an issue accessing `schema.Schema`?

Reply:
I agree, we could use this workaround for now to resolve the blocker but continue to investigate a long term fix with https://github.com/devfile/api/issues/1595.

@Jdubrick WDYT?

Reply:
I agree as well with pushing this through as a temp resolution to the blocker. We should prioritize the long term fix investigation for this upcoming sprint so the temp fix isn't being relied on forever. @michael-valdron @thepetk

Reply:
@michael-valdron @Jdubrick could you please approve? :hugs: